File: Class/1013/hw_login/app/src/main/java/com/example/hw_login/MainActivity.java
package com.example.hw_login;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {

    Button btn;
    EditText id, pwd;
    String ID,PWD;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        id=(EditText)findViewById(R.id.edittext1);
        pwd=(EditText)findViewById(R.id.edittext2);

        btn=(Button)findViewById(R.id.button1);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                ID=id.getText().toString();
                PWD=pwd.getText().toString();
                Intent intent=new Intent(MainActivity.this,SubActivity.class); //현재 화면의 제어권 넘어감
                intent.putExtra("UserID",ID);
                intent.putExtra("Password",PWD);
                startActivity(intent);
            }
        });
    }
}

File: Kotlin/week2/Intent/app/src/main/java/com/example/intent/MainActivity.kt
package com.example.intent

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_a.setOnClickListener{
            val intent = Intent(this,SubActivity::class.java) //다음 화면으로 이동하기 위한 인텐트 객체 생성
            intent.putExtra("msg",tv_sendMsg.text.toString())
            startActivity(intent)
        }
    }
}